platform-docs/docs/basic-concepts/settings.md

```diff
@@ -6,14 +6,43 @@ sidebar_position: 3
 
 You can customize the block editor by providing a `settings` prop to the `BlockEditorProvider` component. This prop accepts an object with the following properties:
 
+## __experimentalFeatures
```

Comment:
Two things here: 

 1- We need to stabilize this setting
 2- I've noticed that by default, if we don't define this setting, the block editor doesn't support any of the default block supports (aside font size for some reason). I think that's not great and we should update the default value (or merge the prop with some sane defaults).

Reply:
Even the label `__experimentalFeatures` is confusing - aren't these values (set [here](https://github.com/WordPress/gutenberg/blob/352ab9993ed4eefd0d11f15dc4146283ec2080ff/lib/block-editor-settings.php#L81C40-L81C69)) merged theme.json settings derived from [gutenberg_get_global_settings()](https://github.com/WordPress/gutenberg/blob/352ab9993ed4eefd0d11f15dc4146283ec2080ff/lib/global-styles-and-settings.php#L104-L104)?

Reply:
yes, I'm curious what would you call them?

Reply:
Good question! This might sound naïve, but `themeSettings` 😃 🤷🏻

Reply:
`themeSettings` makes sense after a quick glance at what's under `__experimentalFeatures` currently.

Would it be a little redundant having the `Settings` suffix for a property directly under `settings`, i.e. `settings.themeSettings`? I don't mind it, but don't have strong opinions here.

Reply:
Yes, that's what's bothering me. we'd have settings.themeSettings, settings.styles and I know that we also want to introduce the "styles" part of theme.json, would that be settings.themeStyles (feels like the current styles are also themeStyles, just a different type).

I don't have solutions, but I think we need to look at this wholistically and keep in mind that block-editor  package is not necessarily WordPress specific code. I think right now there's no concept of "theme" there.